jsapp/js/components/formLanding.es6

```diff
@@ -469,6 +476,10 @@ export class FormLanding extends React.Component {
               {t('Manage Translations')}
             </bem.PopoverMenu__link>
           }
+          <bem.PopoverMenu__link onClick={this.showEncryptionModal}>
```

Comment:
Would it be hard to add this button in the "More actions" dropdown on the list of assets? See how `jsapp/js/components/modalForms/translationSettings.es6` does it :)

Reply:
Took me longer than expected! I think I have a solid understanding of how this works, let me know if I did it right ;)